### DISCUSSION TITLE
Solis hybrid inverter with Pylontech batteries

### DISCUSSION THREAD

**Discussion by Dozer302** [wills106/homeassistant-solax-modbus#550] [General]
Is there a way to control the battery export power using HA ?
For example: EV is plugged in and starts charging on a Zappi.

HA sees the Zappi start pulling power and if greater than 100watts sets the solis battery power limits to 0 so as to not use the battery.

When the EV is charged to full or is drawing less than 100 watts then set the battyer power back to "normal" output power.

This could be done via an automation if I can work what to do...
any ideas ?

**Comment by alienatedsec** [comment]
In theory, you could set different modes on the inverter when Zappi starts pulling from the grid.
